Project/src/main/java/app/models/Review.java

```diff
@@ -0,0 +1,12 @@
+package app.models;
+
+import javax.persistence.*;
+
+@Entity
+public class Review {
```

Comment:
Why do we need an entire class for review when it's just a String and the Article knows it's own reviewer? Alternatively we need to add a String to Review, but just changing the review to a String in Article I think would make more sense

Reply:
I agree, seems redundant